README.md

```diff
@@ -1 +1,201 @@
+- HTML and CSS best practices, Linters documentation, JavaScript.
+
+![Preview Image](Images/preview.png)
+
+
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+
+<details>
+  <summary>Frontend developer</summary>
+  <ul>
+    <li>HTML</li>
+    <li>CSS</li>
+    <li>js</li>
+    <li><a href="https://fonts.googleapis.com"><li>
+  </ul>
+</details>
+
+
+### Key Features <a name="key-features"></a>
+
+- Added index.html file
+- Added style.css file
+- Added index.js file
+- Included Linters
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo](Coming soon)
+- [Live Demo](https://wils-fourr-speed.github.io/Capstone-1/)
+
+<p align="right">(<a href="#">back to top</a>)</p>
+
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+1. A Code Editor like:
+- VSCode
+- Atom, etc.
+
+2. NodeJs extension installed in your code editor.
+
+3. A Web Browser like
+- Google Chrome
+- Firefox
+- Microsoft Edge
+- Opera, etc.
+
+
+### Setup
+
+Clone this repository to your desired folder:
+
+git clone https://github.com/Wils-FOURR-speed/Capstone-1.git
+
+### Install
+
+Install this project with:
+
+Run this command:
+
+...sh
+  cd my-project
+  npm install
+  ...
+
+
+### Usage
+
+To run the project, execute the following command:
+
+Example command:
+Open the index.html using live server.
+
+### Run tests
+
+To run tests, run the following command:
+
+- To trace all linter errors, run
+  ...
+
+  npx hint .
+  ...
+- To check any styling error, run  
+  ...
+
+  npx stylelint "**/*.{css,scss}"
+  ...
+- To trace any JavaScript error, run
+  ...
+
+  npx eslint .
+  ...
+
+
+## 👥 Author <a name="Wilsan"></a>
+
+
+👤 **Wilsan**
+
+- GitHub: [@githubhandle](https://github.com/Wils-FOURR-speed)
+- Twitter: [@twitterhandle](https://twitter.com/WFourrspeed)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/nji-wilsan-ndenge-47b7a826a/)
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 🔭 Future Features <a name="future-features"></a>
+
+
+- [ ] Add more transitions and animations effects
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/Wils-FOURR-speed/Capstone-1/issues).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## ⭐️ Show your support <a name="support"></a>
+
+
+If you like this project, please give me a star.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+Big thanks to:
+- Microverse
+- The original author of this wonderful design, Cindy Shin in Behance.
+
```

Comment:
- [ ] As suggested by the previous reviewer kindly reference the author like this 
`- [Cindy Shin in Behance](https://www.behance.net/adagio07) is credited with the original concept and idea for the design showcased.`